elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -139,6 +167,20 @@ private RealmAddHandler() {
             super(SECURITY_REALM_RUNTIME_CAPABILITY, ATTRIBUTES);
         }
 
+        private SecretKey getKey(OperationContext context, String credentialStoreReference, String alias) throws OperationFailedException{
+            ExceptionFunction<OperationContext, CredentialStore, OperationFailedException> credentialStoreApi = context.getCapabilityRuntimeAPI(CREDENTIAL_STORE_API_CAPABILITY, credentialStoreReference, ExceptionFunction.class);
+            CredentialStore credentialStoreResource = credentialStoreApi.apply(context);
+            try {
+                SecretKeyCredential credential = credentialStoreResource.retrieve(alias, SecretKeyCredential.class);
+                if (credential == null) {
+                    throw ROOT_LOGGER.credentialDoesNotExist(alias, "SecretKey");
```

Comment:
Should use `SecretKeyCredential.class.getSimpleName()` instead of "SecretKey".

Reply:
Updated